src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -3484,6 +3448,80 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 							}
 							break;
 
+						//
+						// static GetRuntimeEvent (this Type type, string name)
+						// static GetRuntimeField (this Type type, string name)
+						// static GetRuntimeMethod (this Type type, string name, Type[] parameters)
+						// static GetRuntimeProperty (this Type type, string name)
+						//
+						case var getRuntimeMember when (getRuntimeMember.StartsWith ("GetRuntime")
+							&& calledMethod.DeclaringType.Namespace == "System.Reflection"
+							&& calledMethod.DeclaringType.Name == "RuntimeReflectionExtensions"
+							&& calledMethod.Parameters.Count >= 2
+							&& calledMethod.Parameters [0].ParameterType.FullName == "System.Type"
+							&& calledMethod.Parameters [1].ParameterType.FullName == "System.String"): {
+
+								reflectionContext.AnalyzingPattern ();
+								DynamicallyAccessedMemberKinds? memberKind = null;
+
+								switch (getRuntimeMember) {
+									case var mt when getRuntimeMember.EndsWith ("Event"):
+										memberKind = DynamicallyAccessedMemberKinds.Events;
+										break;
+
+									case var mt when getRuntimeMember.EndsWith ("Field"):
+										memberKind = DynamicallyAccessedMemberKinds.Fields;
```

Comment:
I tried the runtime behavior and it only returns public members via these calls. Interestingly enough `GetRuntimeFields` (the plural) returns all private/public. But the `GetRuntimeField` returns only public ones. Probably similar for the rest of these.